v0/destinations/am/transform.js

```diff
@@ -206,6 +207,8 @@ function responseBuilderSimple(
       endpoint = ENDPOINT;
       // event_type for identify event is $identify
       rawPayload.event_type = EventType.IDENTIFY_AM;
+      rawPayload.country = get(message, "context.location.country");
+      rawPayload.city = get(message, "context.location.city");
```

Comment:
Let's not take `country` and `city` fields from `context.location` as they are not part of our common API spec.